### ISSUE TITLE
SIAP Table Figure size

### ISSUE THREAD

**Issue by Qgel** [dstl/Stone-Soup#438]
Hello,

When generating the SIAP Table using the `SIAPTableMetricsGenerator` as in the `Metrics` example, the figure that is plotted is cut off and only shows a small part of the table (this is the whole plot, including the offset of the table): 

![grafik](https://user-images.githubusercontent.com/1146690/116109373-fdf5ca80-a6b4-11eb-8c00-b18b688e9e86.png)

If I remove the following two lines starting in `metrictables.py:71`

```
        fig = plt.figure(figsize=(20, 1))
        fig.subplots_adjust(left=0.4, top=0.6)
```

 and set the figure size to `(12, 8)` via `rcParams` I get plot of the table as expected.

Best Regards,
Mirko

**Comment by sdhiscocks** [comment]
Sorry @Qgel I only just spotted this issue. I just replied in #457 which raised same issue: addding `bbox_inches='tight'` to the `savefig` arguments should resolve it.

**Comment by sdhiscocks** [comment]
@Qgel Just merged #459. Could you confirm if this solves the issue?

**Comment by sdhiscocks** [comment]
Assume resolved.
